bitnami/mysql/8.0/debian-11/rootfs/opt/bitnami/scripts/libmysql.sh

```diff
@@ -178,6 +178,65 @@ pid_file=${DB_PID_FILE}
 EOF
 }
 
+########################
+# Make a dump on master database and update slave database
+# Globals:
+#   DB_*
+# Arguments:
+#   None
+# Returns:
+#   None
+#########################
+mysql_exec_initial_dump() {
+    info "MySQL dump master data start..."
+    
+    mysql -h "$DB_MASTER_HOST" -P "$DB_MASTER_PORT_NUMBER" -u "$DB_MASTER_ROOT_USER" -p"$DB_MASTER_ROOT_PASSWORD" -e 'RESET MASTER;'
```

Comment:
I have one last question about this. This would execute `RESET MASTER` for every slave. Is this necessary? 
Note that RESET MASTER is already executed in [mysql_initialize() ](https://github.com/bitnami/containers/blob/f81b69b67f0bd59bd3d1a09a3f141938722456f8/bitnami/mysql/8.0/debian-11/rootfs/opt/bitnami/scripts/libmysql.sh#L309). Could you please confirm?

Reply:
Hi @dgomezleon.
Sorry about my lazy response.. I was on vacation and I returned in the last days.
Your question is very interesting and makes me think about how replication works. So let me explain my point of view.

I have a master setup and a slave was broken because of a crash on the slave server.. So, I make a reinstallation of the new slave server and when I up the MySQL slave server this [error](https://github.com/bitnami/containers/issues/54028) occurs and this error is because the Mysql master server is missing some binary files. To fix it, I could make a manual mysql dump on master and update the slave, but to do it I must login in recovery mode on MySQL slave because the mysql.user could not be transferred by binary files and we do not have any valid user to login. Analyzing this scenario, I make this approach to make an auto mysql dump on master and import to a slave database.
 
Analyzing your question about the mysql_initialize(), this is only exec on the first up of the container. In my scenario, the reset will not be executed and the mysql master binary files are not modified. To resolve this, I could manually get mysql dump on the master server, make the reset master and run the mysql dump in slave. 

So, when the MySQL master server doesn't have all the binary files, for each slave that is added to the server the reset master will be necessary for the slave to start replicating at the point of the master server stop.

One important point that your question made me think about is: that is very important that all slave servers have the last data present in the master database, when we restart replication after reset master the slave has the same data and does not miss any data. So, I think is important to add one alert to make sure that all slaves have all data updated, before start the setup the new slave.

I don't know if it explains my point of view. But I'm open to having some discussion about this and making the better to the package.